src/families/tezos/bridge/libcore.js

```diff
@@ -132,6 +132,10 @@ const getTransactionStatus = async (a, t) => {
     );
   }
 
+  if (estimatedFees.gt(a.balance)) {
```

Comment:
to me we should do that if:
- there is no errors.amount already
- it's a "subAccuont" transaction 🤔

Reply:
current version should do that